1_fetch.smk

```diff
@@ -0,0 +1,43 @@
+import os
+
+configfile: "1_fetch/fetch_config.yaml"
+
+
+def get_mntoha_input_files(wildcards):
+    '''List all MNTOHA files to download
+    Includes metadata, temperature observations, and drivers
+
+    :param wildcards: Snakemake wildcards (unused)
```

Comment:
Isn't `wildcards` used in line 40?

Reply:
Line 40 is outside the scope of `get_mntoha_input_files`, so that `wildcards` is a different variable. We could rename one for clarity.